lib/feedjira/feed.rb

```diff
@@ -108,8 +108,8 @@ def parse_last_modified(response)
         lm = response.headers['last-modified']
         DateTime.parse(lm).to_time
       rescue StandardError => e
-        Feedjira::Logger.warn { "Failed to parse last modified '#{lm}'" }
-        Feedjira::Logger.exception(e)
+        Feedjira.logger.fatal("Failed to parse last modified '#{lm}'")
+        Feedjira.logger.fatal(e)
         nil
```

Comment:
I just don't have a ruby environment here. Will Logger.fatel(e) also format the exeption and spill out the StackTrace?